### PR TITLE
Update centos-redhat config section

### PR DESCRIPTION
**Pull request by dulanism**
Updated CentOS/Redhat section for systemd user; more commonly used.

### CODE REVIEW

File: chapter-installation.asciidoc [hunk 1]
```diff
@@ -251,38 +251,59 @@ only.
 [[service-linux]]
 ==== Setting up the Service on Linux
 
-You can configure the repository manager to run the +nexus+ script within +etc/init.d+. +etc/init.d+ is a 
-standard service management package for Linux, a program that contains a console of start, stop, and restart 
-commands. +/etc/init.d+ files are essentially shell scripts that load commands to manage the nexus service, 
-triggered by a symlink. On a Linux system perform the operations as the root user are listed below.
+You can configure the repository manager to run as a service within either +etc/init.d+ or +systemd+. 
+Both +etc/init.d+ and +systemd+ are service management packages used in Linux-based systems such as Ubuntu and 
+CentOS. Both are daemons that contain a console of start, stop, and restart commands. +/etc/init.d+ and +systemd+
```
Comment (mosabua) [inline chapter-installation.asciidoc]: They are not daemons and dont contain a console. The are system startup frameworks that manage the boot process and therefore starting of daemons..

File: chapter-installation.asciidoc [hunk 2]
```diff
@@ -249,77 +249,53 @@ follow system administration best practice and use a service specific user with
...
-For Ubuntu and Debian, the equivalent to the +chkconfig+ tool is +update-rc.d+. Run a default priority to add the +nexus+ service before starting it.
-
+Activate the service with the following commands:
 ----
-cd /etc/init.d
-sudo update-rc.d nexus defaults
-sudo service nexus start
+sudo systemctl daemon-reload
+sudo systemctl enable nexus.service
+sudo wystemctl start nexus.service
```
Comment (mosabua) [inline chapter-installation.asciidoc]: wystemctl .. typo!

File: chapter-installation.asciidoc [hunk 3]
```diff
@@ -249,77 +249,53 @@ follow system administration best practice and use a service specific user with
...
 ----
 
-After changes to the files above, symlink +$NEXUS_HOME/bin/nexus+ to +/etc/init.d/nexus+:
+[[run-service]]
+===== Running the Service
 
-----
-sudo ln -s $NEXUS_HOME/bin/nexus /etc/init.d/nexus
-----
+This example script uses +systemd+ to run the repository manager service. Create a file called +nexus.service+.
```
Comment (mosabua) [inline chapter-installation.asciidoc]: Where do you create that file?

Comment (dulanism) [inline chapter-installation.asciidoc]: /usr/local/bin. From what I read when testing that would be an ideal place to store scripts that are run by a systemd daemon/service

**Pull request merged**